### PR TITLE
HTM-1389: add test data for search filtering

### PR DESCRIPTION
**Pull request by mprins**
[HTM-1389](https://b3partners.atlassian.net/browse/HTM-1389) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=Tailormap&utm_campaign=badge_info)

relates to https://github.com/Tailormap/tailormap-viewer/pull/787

[HTM-1389]: https://b3partners.atlassian.net/browse/HTM-1389?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/java/org/tailormap/api/configuration/dev/PopulateTestData.java [hunk 1]
```diff
@@ -978,6 +978,12 @@ public void createAppTestData() throws Exception {
             .setAuthorizationRules(ruleAnonymousRead)
             .setComponents(
                 List.of(
+                    new Component()
+                        .type("SIMPLE_SEARCH")
+                        .config(
+                            new ComponentConfig()
+                                .enabled(true)
+                                .putAdditionalProperty("municipalities", List.of("Utrecht"))),
```
Comment (geertplaisier) [inline src/main/java/org/tailormap/api/configuration/dev/PopulateTestData.java:986]: We will be using municipality code for the list, so should be `List.of("0344")`, see https://github.com/Tailormap/tailormap-viewer/pull/787

File: src/test/java/org/tailormap/api/controller/AppControllerIntegrationTest.java [hunk 2]
```diff
@@ -45,9 +45,12 @@ void returns_default_when_no_arguments() throws Exception {
         .andExpect(jsonPath("$.name").value("default"))
         .andExpect(jsonPath("$.title").value("Tailormap demo"))
         .andExpect(jsonPath("$.components").isArray())
-        .andExpect(jsonPath("$.components.length()").value(2))
-        .andExpect(jsonPath("$.components[0].type").value("EDIT"))
+        .andExpect(jsonPath("$.components.length()").value(3))
+        .andExpect(jsonPath("$.components[0].type").value("SIMPLE_SEARCH"))
         .andExpect(jsonPath("$.components[0].config.enabled").value(true))
+        .andExpect(jsonPath("$.components[0].config.municipalities").value("Utrecht"))
```
Comment (geertplaisier) [inline src/test/java/org/tailormap/api/controller/AppControllerIntegrationTest.java:51]: `.andExpect(jsonPath("$.components[0].config.municipalities").value("0344"))`

**Review by geertplaisier — CHANGES_REQUESTED** [review]

File: src/main/java/org/tailormap/api/configuration/dev/PopulateTestData.java (hunk 1 above)
Comment (mprins) [inline src/main/java/org/tailormap/api/configuration/dev/PopulateTestData.java:986]:
```suggestion
                                .putAdditionalProperty("municipalities", List.of("0344"))),
```

File: src/test/java/org/tailormap/api/controller/AppControllerIntegrationTest.java (hunk 2 above)
Comment (mprins) [inline src/test/java/org/tailormap/api/controller/AppControllerIntegrationTest.java:51]:
```suggestion
        .andExpect(jsonPath("$.components[0].config.municipalities").value("0344"))
```

**Pull request merged**